stdlib/public/runtime/MetadataLookup.cpp

```diff
@@ -1159,8 +1167,8 @@ class DecodedMetadataBuilder {
   BuiltType createFunctionType(
                            ArrayRef<Demangle::FunctionParam<BuiltType>> params,
                            BuiltType result, FunctionTypeFlags flags) const {
-    std::vector<BuiltType> paramTypes;
-    std::vector<uint32_t> paramFlags;
+    SmallVector<BuiltType, 8> paramTypes;
+    SmallVector<uint32_t, 8> paramFlags;
 
     // Fill in the parameters.
     paramTypes.reserve(params.size());
```

Comment:
Just checking, the reserve is still fine after?

Reply:
Yes, should be fine